client/my-sites/upgrades/checkout/supporting-text.jsx

```diff
@@ -6,6 +6,7 @@ var React = require( 'react' );
  * Internal dependencies
  */
 var cartValues = require( 'lib/cart-values' ),
+	support = require( 'lib/support' ),
```

Comment:
It happens that we already have a [`url` module](https://github.com/Automattic/wp-calypso/tree/master/client/lib/url). I would suggest using it since there is no need to reinvent the wheel. Moreover you'll have nicer imports:

```javascript
import url from 'lib/url';

var refundDocsLink = (
  <a className="credit-card-supporting-text__refund-link"
    href={ url.REFUNDS }
    target="_blank" />
);
```